core/include/msg.h

```diff
@@ -355,6 +355,17 @@ int msg_reply(msg_t *m, msg_t *reply);
  */
 int msg_reply_int(msg_t *m, msg_t *reply);
 
+/**
+ * @brief Check how many messages are available in the message queue of a
```

Comment:
```suggestion
 * @brief Check how many messages are available (waiting) in the message queue of a
```

Reply:
Will fix, and for `msg_avail()` as well.